### ISSUE TITLE
[7.0.1] No .d.ts for puppeteer-core

### ISSUE THREAD

**Issue by thomassuckow** [puppeteer/puppeteer#6820]
### Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 7.0.1
* Platform / OS version: Windows 10
* Node.js version: 15.0.1

**What steps will reproduce the problem?**

Install puppeteer-core and attempt to use it from typescript

**What is the expected result?**

It works

**What happens instead?**

```
TS7016: Could not find a declaration file for module 'puppeteer-core'. '[...]\puppeteer-core\cjs-entry-core.js' implicitly has an 'any' type.
```

It appears that while cjs-entry.js has a matching cjs-entry.d.ts, cjs-entry-core.js does not.

**Comment by mathiasbynens** [comment]
Jack, could you PTAL?

**Comment by danielrentz** [comment]
Same for me, just tried to update from 5.5 to 7.0.1, and `import { ... } from "puppeteer"` stopped working.

How is that supposed to work? README still says to use the `@types/puppeteer` package, but that does not work either.

**Comment by thomassuckow** [comment]
@danielrentz You are probably more interested in #6816 for type issues. This is specifically regarding `puppeteer-core` instead of `puppeteer`

**Comment by danielrentz** [comment]
Right, thanks for the pointer!

**Comment by thomassuckow** [comment]
#6837 should resolve this once merged/released

**Comment by thomassuckow** [comment]
Fixed in 7.0.4
